### ISSUE TITLE
Setting font-family on SVG text element has no effect 

### ISSUE THREAD

**Issue by eNepper** [lovell/sharp#1549]
I'm unable to change the font of the text in the SVG. 
I tried to run FC_DEBUG and it looks like no matter what is selects Liberation Serif.

I have fontconfig version 2.12.6 installed on my system.

This is the SVG I'm using 
`<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 108 16" version="1.1"><text font-family="sans-serif" font-size="12px" x="54" y="12" fill="#000000" text-anchor="middle" >Fancy Text</text></svg>`

```
'use strict';
const sharp = require('sharp');
const fs = require('fs');

let buffer = '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 108 16" version="1.1"><text font-family="sans-serif" font-size="12px" x="54" y="12" fill="#000000" text-anchor="middle" >Fancy Text</text></svg>';

sharp(Buffer.from(buffer))
    .toBuffer().then(buffer => {
        fs.writeFile('Sans-serif.png', buffer, (err) => { });
});
```

If I view the same SVG in the browser it looks correct.

-- Esben N.

**Comment by eNepper** [comment]
I have tested it on two separate Ubuntu 18.04 machines now and the output is identical.

**Comment by lovell** [comment]
Hello, are you using the pre-built binaries? What's the output of `console.log(sharp.versions)`?

**Comment by lovell** [comment]
Thank you. I tried the offending SVG image with the prebuilt binaries on an Ubuntu 18.10 machine, which appears to correctly select tthe `/usr/share/fonts/truetype/dejavu/DejaVuSans.ttf` font from the `fonts-dejavu-core` package.

Are you able to provide a Docker-based image in which this problem is reproducible?

**Comment by eNepper** [comment]
You can find a sample on docker here: https://github.com/eNepper/svgText
To see the result I copy the files out of the container 

Run:
docker build . -t nodejs:svg
docker run -it nodejs:svg /bin/bash

Copy: 
docker cp [container]:/usr/src/app/img/ .

**Comment by lovell** [comment]
Perfect, thank you, it looks like you've run into https://gitlab.gnome.org/GNOME/librsvg/issues/319 that was fixed in librsvg v2.44.0. You can workaround this for now by wrapping the font-family names in quotes.

```svg
<text font-family="'sans'" ...
```

**Comment by eNepper** [comment]
I have tested your solution and can confirm that it works.
Thanks for the help!

**Comment by lovell** [comment]
sharp v0.22.0 now available, providing libvips v8.7.4 and librsvg v2.45.5.

**Comment by sgwanlee** [comment]
In my case, it worked after I change the font name as the same as the name I installed on my mac.
